tests/test_m006_return3e_pgstore_noddl_pasay_runtime.py

```
@@ -0,0 +1,283 @@
+    st = _install_fake_psycopg2_env(deny_create=True)
+    with patch.object(_pg, "connect", st["fake_connect"]):
+        store = StateStore("unused", runtime_mode="cloudflare-container")
+
+    assert isinstance(store, PostgresStateStore)
+    executed = st["executed_sql"]
+    any_create = any(
+        s.lower().startswith(("create ", "alter ", "drop ", "grant ", "revoke "))
+        for s in executed
+    )
+    assert not any_create, (
+        f"M006 FAIL: PostgresStateStore issued DDL through pasay_runtime.\n"
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)